servlet/5.0/_index.md

```diff
@@ -10,7 +10,8 @@ Jakarta Servlet defines a server-side API for handling HTTP requests and respons
 * [Jakarta Servlet 5.0 Specification Document](./jakarta-servlet-spec-5.0.pdf) (PDF)
 * [Jakarta Servlet 5.0 Specification Document](./jakarta-servlet-spec-5.0.html) (HTML)
 * [Jakarta Servlet 5.0 Javadoc](./apidocs)
-* [Jakarta Servlet 5.0 TCK](https://download.eclipse.org/jakartaee/servlet/5.0/jakarta-servlet-tck-5.0.1.zip)([sig](https://download.eclipse.org/jakartaee/servlet/5.0/jakarta-servlet-tck-5.0.1.zip.sig),[sha](https://download.eclipse.org/jakartaee/servlet/5.0/jakarta-servlet-tck-5.0.1.zip.sha256),[pub](https://raw.githubusercontent.com/jakartaee/specification-committee/master/jakartaee-spec-committee.pub))
+* [Jakarta Servlet 5.0.1 TCK](https://download.eclipse.org/jakartaee/servlet/5.0/jakarta-servlet-tck-5.0.1.zip)  ([sig](https://download.eclipse.org/jakartaee/servlet/5.0/jakarta-servlet-tck-5.0.1.zip.sig),  [sha](https://download.eclipse.org/jakartaee/servlet/5.0/jakarta-servlet-tck-5.0.1.zip.sha256),  [pub](https://raw.githubusercontent.com/jakartaee/specification-committee/master/jakartaee-spec-committee.pub))
```

Comment:
Same comment here...  I know there was a 5.0.0 TCK that was used for the original 9.0 TCK.  When we produced the 9.0.1 TCK, there was a corresponding Servlet 5.0.1 TCK.  So, I think there should be a top-level 5.0.0 TCK with two sub-bullets for 5.0.1 and 5.0.2 TCKs.

Reply:
PR log review shows: 
[Servlet TCK 5.0.1 update with excludes for accepted TCK challenges https://github.com/eclipse-ee4j/servlet-api/issues/385 + https://github.com/eclipse-ee4j/servlet-api/issues/378 #323](https://github.com/jakartaee/specifications/pull/323) -- Jan 10 -- **Replaced** Servlet TCK 5.0.0 with 5.0.1. The two issues in the title describe the test challenges:

[servlet-api #385](https://github.com/eclipse-ee4j/servlet-api/issues/385) and 
[servlet-api #378](https://github.com/eclipse-ee4j/servlet-api/issues/378)

This PR appears to have been paired with Platform TCK update PR #322 which also replaced the Platform TCK link.

Strictly speaking, these PRs should have ADDED the new TCKs, but we weren't clear on that. I did not attempt to rectify these prior decisions. We certainly could if you and/or the committee things that would be helpful.

Reply:
All that I am indicating is that we should list the 5.0.0 TCK at the top level, with the 5.0.1 and 5.0.2 TCKs as sub-bullets.  I think that completes the picture for the TCKs, doesn't it?  Just pretending that the 5.0.0 TCK never existed doesn't seem right.

Reply:
Please see the latest commit.